app/assets/config/manifest.js

```diff
@@ -1,3 +1,4 @@
 //= link_tree ../images
 //= link_directory ../javascripts .js
```

Comment:
Should we be removing `//= link_directory ../javascripts .js`, like we do for the CSS line?

Reply:
yes, well spotted, thank you